src/shared/lib/tracking.ts

```diff
@@ -37,14 +37,18 @@ export function initializeTracking() {
 }
 
 export function trackEvent(event: GAEvent) {
-    getGAInstance()(
-        'send',
-        'event',
-        event.category,
-        event.action,
-        event.label,
-        event.fieldsObject
-    );
+    // trackEvent is used to send custom UI events which may depend on custom configuration within GA
+    // for other installers, we want to shut these off, leaving them with bare-bones out-of-box GA implemenation
+    if (/cbioportal\.org$/.test(getBrowserWindow().location.host)) {
```

Comment:
@inodb i think this should catch all our portals, or at least those we are tracking with GA. whachu think?

Reply:
maybe also track *.mskcc.org portals?

Reply:
@inodb good catch! updated